File: front/small-box/src/app/models/supplyCorrectionNoteDto.ts
import { SupplyReportDto } from "./suppplyReportDto";

export class SupplyCorrectionNote{
    supplyDate!:Date;
    supplyExcerciseYear!:number;
    supplyNumber!:number;
    from!:string;
    to!:string;
    supplyReport!:Array<SupplyReportDto>;
    depositName!:string;

}

File: src/main/java/com/lord/small_box/dtos/SupplyCorrectionNoteDto.java
package com.lord.small_box.dtos;

import java.util.Calendar;
import java.util.List;

import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
public class SupplyCorrectionNoteDto {

	private Calendar supplyDate;
	
	private int supplyNumber;
	
	private int supplyExcerciseYear;
	
	private String from;
	
	private String to;
	
	private List<SupplyReportDto> supplyReport;
	
	private String depositName;
}
